tests/migration/migration.go

```diff
@@ -269,12 +271,22 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 		})
 		Context("[Serial] with bandwidth limitations", Serial, func() {
 
-			var repeatedlyMigrateWithBandwidthLimitation = func(vmi *v1.VirtualMachineInstance, bandwidth string, repeat int) time.Duration {
+			updateMigrationPolicyBandwidth := func(migrationPolicy *v1alpha1.MigrationPolicy, bandwidth resource.Quantity) {
+				migrationPolicy, err = virtClient.MigrationPolicy().Get(context.Background(), migrationPolicy.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				if policyBandwidth := migrationPolicy.Spec.BandwidthPerMigration; policyBandwidth != nil && policyBandwidth.Cmp(bandwidth) == 0 {
```

Comment:
`policyBandwidth.Equal(bandwidth)`

Reply:
Wow, cool! Is this function new?
Thanks for pointing it out! :)

Reply:
Well, looks like it has been available since `1.16`. I'm surprised too :D